build.gradle

```diff
@@ -79,7 +79,7 @@ dependencies {
     // This dependency is found on compile classpath of this component and consumers.
     compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.3.4'
     compile 'com.google.guava:guava:23.0'
-    compile ('io.left.rightmesh:rightmesh-java-library-dev:0.4.2-IGOR-SNAPSHOT')
+    compile ('io.left.rightmesh:rightmesh-java-library:0.4.2')
```

Comment:
We have a `0.4.3` now. Would it make sense to hold off until the merge into the library when we'll bump to `1.0`?

Reply:
make sense

Reply:
Speaking of - has the library side of this stuff merged in already?

Reply:
Yep sounds like it has - we can bump to 0.4.3 here.

Reply:
Okay. Somehow I had rattling around in memory that we were bumping the library to 1.0 once payment channels were in. :upside_down_face: So just 0.4.3 then.